theme/scripts/cart.ts

```diff
@@ -284,27 +286,53 @@ export default(function() {
         }
       })
     },
+  
+    handleProductAddedToCart() {
+      const desktopProductAddToCartButton = document.querySelector(Cart.SELECTORS.desktopProductAddToCartButton);
+      const mobileProductAddToCartButton = document.querySelector(Cart.SELECTORS.mobileProductAddToCartButton);
 
-    //TO-DO (#178): This function should check when the add to cart button is clicked and use a setTimeout to change the text to "Added to cart" and then revert back to "Add to cart" after a few seconds.
-    handleAddToCartButton(variantId: string, action: Action) {
-      const addToCartButton = document.querySelector(Cart.SELECTORS.addToCartButton);
-      
-      if (!addToCartButton) return;
-      
-      const addToCartVariantId = addToCartButton.getAttribute(Cart.ATTRIBUTES.variantId);
+      if (desktopProductAddToCartButton) {
+        Cart.handleProductAddedToCartButtonStyling(desktopProductAddToCartButton);
+      }
 
-      if (addToCartVariantId === variantId) {
-        if (action === Action.ADD) {
-          addToCartButton.textContent = Cart.CONSTANTS.addedToCart;
-          addToCartButton.setAttribute(Cart.ATTRIBUTES.disabled, Cart.ATTRIBUTES.disabled);
-        }
+      if (mobileProductAddToCartButton) {
+        Cart.handleProductAddedToCartButtonStyling(mobileProductAddToCartButton);
+      }
+    },
+    
+    handleProductAddedToCartButtonStyling(button: Element) {
+      button.textContent = Cart.CONSTANTS.addedToCart;
+      button.setAttribute(Cart.ATTRIBUTES.disabled, Cart.ATTRIBUTES.disabled);
+
+      setTimeout(() => {
+        button.textContent = Cart.CONSTANTS.addToCart;
+        button.removeAttribute(Cart.ATTRIBUTES.disabled);
+      }, 2000);  
+    },
 
-        if (action === Action.REMOVE) {
-          addToCartButton.textContent = Cart.CONSTANTS.addToCart;
-          addToCartButton.removeAttribute(Cart.ATTRIBUTES.disabled);
-        }
+    //Handles any errors adding an item to the cart
+    handleProductAddedToCartError() {
+      const desktopProductAddToCartButtonError = document.querySelector(Cart.SELECTORS.desktopProductAddToCartButtonError);
+      const mobileProductAddToCartButtonError = document.querySelector(Cart.SELECTORS.mobileProductAddToCartButtonError);
+
+      if (desktopProductAddToCartButtonError) {
+        Cart.handleProductAddedToCartButtonErrorStyling(desktopProductAddToCartButtonError);
       }
+
+      if (mobileProductAddToCartButtonError) {
+        Cart.handleProductAddedToCartButtonErrorStyling(mobileProductAddToCartButtonError);
+      }
+    },
+
+    //Shows text below the Add to Cart button, indicating that the customer was not able to add that item to their cart
+    handleProductAddedToCartButtonErrorStyling(button: Element) {
+      button.classList.toggle("none");
```

Comment:
Maybe we could animate this in instead using opacity, and adding a subtle transition in css